# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1328206115 28800
#      Thu Feb 02 10:08:35 2012 -0800
# Branch stable
# Node ID 0216b5d6c80c0e19ac4db9d8d61c717b0d96c4d0
# Parent  99428221b4e12873fd8360c002cd48aa2b6ab5b4
semilogx.m: Correct incorrectly coded %!test.

* semilogx.m: Correct incorrectly coded %!test.

diff --git a/scripts/plot/semilogx.m b/scripts/plot/semilogx.m
--- a/scripts/plot/semilogx.m
+++ b/scripts/plot/semilogx.m
@@ -107,17 +107,17 @@ endfunction
 %!   assert (get (gca, "yscale"), "linear");
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
 
 %!test
 %! hf = figure ("visible", "off");
 %! unwind_protect
-%!   a = logspace (-5, 1, 10);
-%!   b =-logspace (-5, 1, 10);
+%!   a =-logspace (-5, 1, 10);
+%!   b = logspace (-5, 1, 10);
 %!   semilogx (a, b);
 %!   axis tight;
-%!   assert (all (get (gca, "ytick") < 0));
+%!   assert (all (get (gca, "xtick") < 0));
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
 
